spec/federation_messages/federation_messages_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "spec_helper"
+require "federation-testbed"
+
+describe "Generation and dispatch of federation messages" do
```

Comment:
I would prefer spec/integration/federation or something like this. "testbed" is a tool to test federation. And the folder name should represent what we test but not the tool we use to test.